UnitsNet/Scripts/UnitDefinitions/Frequency.json

```diff
@@ -16,7 +16,19 @@
                 }
             ]
         },
-        {
+		{
+            "SingularName": "RadianPerSecond",
```

Comment:
This should be put in RotationalSpeed.json, shouldn't it?

From google:
> The radian per second (symbolized rad/s or rad/sec) is the Standard International ( SI ) unit of angular (rotational) speed. This quantity can be defined in either of two senses: average or instantaneous.

Wiki: https://en.wikipedia.org/wiki/Radian_per_second